plugin-tester-java/src/main/protobuf/helloworld.proto

```diff
@@ -11,8 +11,13 @@ service GreeterService {
     // Sends a greeting
     rpc SayHello (HelloRequest) returns (HelloReply) {}
 
+    // Comment spanning
+    // on several lines
```

Comment:
I initially added these thinking I would could test non-simple comments, but I realized there was nothing at the moment in test suite that could allow to inspect metadata of the generated source files. I left it here as a passive documentation.